### ISSUE TITLE
zoom out to show nearest quest if nearby

### ISSUE THREAD

**Issue by matkoniecz** [westnordost/StreetComplete#642]
Currently: New users opens app, gives location permission, app zooms in, quest loading message appears. Then nothing changes (at least in cases where downloaded quest is not next to the current location).

Maybe it would be preferable to zoom out to slow nearest quest, once quest download ends? And generally, if no quests are nearby maybe it would be preferable to show where is the nearest one?

Obviously, if showing nearest quest would zoom out to much (because it is too far away) it would be preferable to trigger quest download.

(Tested as a beta tester on Google play)

**Comment by rugk** [comment]
I think such a unexpected zooming is bad. Better just show an arrow pointing to the direction where the nearest quest is.

**Comment by westnordost** [comment]
This is blocked by https://github.com/tangrams/tangram-es/issues/1371

**Comment by rugk** [comment]
Can we change the issue title of this issue to reflect that this is about showing a _marker_ at the border of the screen and not jumping around?

**Comment by matkoniecz** [comment]
> this is about showing a marker at the border

No, it is about zooming out so at least one marker will be visible

**Comment by rugk** [comment]
Okay, then we need a new issue: https://github.com/westnordost/StreetComplete/issues/871

So @westnordost are both issues blocked by that Mapzen support or what?

**Comment by ENT8R** [comment]
BTW: it is not Mapzen but Tangram...

**Comment by westnordost** [comment]
Hmm, actually what I meant is that #871 is blocked by https://github.com/tangrams/tangram-es/issues/1371 and this would be a will-not-fix then.

**Comment by rugk** [comment]
So add the "wontfix" label…
